x12/parsing.py

```diff
@@ -275,15 +277,24 @@ def parse(
             self._context.reset_transaction()
             return model
 
-    def __init__(self, x12_delimiters: X12Delimiters) -> NoReturn:
+    def __init__(
+        self,
+        transaction_model: X12SegmentGroup,
+        loop_parsers: Dict,
+        x12_delimiters: Optional[X12Delimiters] = None,
+    ) -> NoReturn:
```

Comment:
Should '__init__' have a return type of None? https://stackoverflow.com/questions/46779046/correct-type-annotation-for-init
Looks like NoReturn is reserved for functions that never return or that always throw an exception.

Reply:
Hey @ccorley - Thanks! That's a good catch. 
Reference - https://docs.python.org/3/library/typing.html#typing.NoReturn
You know I think I've misused this type hint consistent throughout the project. I'll correct that and will push up changes.